src/coord/src/util.rs

```diff
@@ -38,3 +44,16 @@ impl<T> Drop for ClientTransmitter<T> {
         }
     }
 }
+
+/// Generates a populated catalog appropriate for tests.
+pub fn generate_test_catalog(path: &Path) -> Result<Catalog, Box<dyn Error>> {
+    let config = Config {
+        path,
+        enable_logging: true,
+        experimental_mode: Some(false),
+        cache_directory: None,
+        build_info: &DUMMY_BUILD_INFO,
+    };
+    let (catalog, _) = Catalog::open(config)?;
+    Ok(catalog)
+}
```

Comment:
to facilitate this module being public, can this just move into `coord/tests/parameters.rs`? We can always make a `utils.rs` mod in the `tests` directory if we need to share it between multiple test files down the road.